validation/jinja2/validation/segment_details.html

```diff
@@ -45,11 +6,13 @@
 <div class="table-responsive">
 
     <div>
-        <button class="button button--success button--back" onclick='goBack()'>Back</button>
+        <button data-cy="back-button" class="button button--success button--back">Back</button>
```

Comment:
This is a _really_ pendatic note, but this should be an `<a>` with the back page as its `href`. If you know what page it should go "back" to in advance, that `href` can be hardcoded. Otherwise, this answer from StackOverflow is really nice:

```javascript
// adapted from: https://stackoverflow.com/a/46163215/6626414
for (let el of document.querySelectorAll('a.button--back')) {
    el.setAttribute('href', document.referrer);

    // this remains the same!
    el.addEventListener('click', (evt) => {
        window.history.back();
    })
}

```